data/remnant/remnant 1 introduction.txt

```diff
@@ -2031,12 +1987,13 @@ mission "Remnant: S-970 Regenerator"
 
 
 
-mission "Remnant: taely intro log"
+mission "Remnant: Taely Intro Log"
 	invisible
 	source
 		government "Remnant"
 	to offer
 		"remnant met taely" >= 1
+		not "Remnant: taely intro log: offered"
```

Comment:
If we don't care about giving some people two of the same log, we don't need to prevent this mission from offering if the old one already offered.
```suggestion
```